src/main/java/dev/joopie/jambot/music/command/PlayCommandHandler.java

```diff
@@ -68,6 +74,42 @@ public RestAction<?> handle(final CommandInteraction event) {
         try {
             final var input = inputOption.getAsString();
 
+            if (input.contains(SPOTIFY_URL)) {
+                if (input.contains("track")) {
+                    String trackId = input.replaceAll("\\?.*", "").substring(input.lastIndexOf("/") + 1);
+                    String spotifyresult = apiSpotifyService.getTrack(trackId);
+
+                    if (spotifyresult.isEmpty()) {
+                        return event.getHook()
+                                .sendMessage("That's bad! We couldn't found anything with the given Spotify URL. Did you copy the right link?");
+
+                    }
+                    final var result = apiYouTubeService.searchForSong(spotifyresult);
+
+                    if (result.isFound()) {
+                        musicService.play(event.getGuild(), event.getUser(), result.getVideoId());
+                        return event.getHook()
+                                .sendMessageEmbeds(createMessageEmbedOfSearchTrack(event.getGuild().getName(), result));
+                    }
+                } else if (input.contains("playlist")) {
+                    AtomicInteger counter = new AtomicInteger();
+                    String playListId = input.replaceAll("\\?.*", "").substring(input.lastIndexOf("/") + 1);
+                    List<String> spotifyresult = apiSpotifyService.getTracksFromPlaylist(playListId);
```

Comment:
```suggestion
                    apiSpotifyService.getTracksFromPlaylist(playListId)
                        .forEach(....);
```

Or a for-each loop is fine too.